tests_and_analysis/test/script_tests/test_dispersion.py

```diff
@@ -17,12 +18,13 @@
 pytestmark = pytest.mark.matplotlib
 # Allow tests with matplotlib marker to be collected and
 # deselected if Matplotlib is not installed
-try:
+
+# Required for mocking
+with suppress(ModuleNotFoundError):
```

Comment:
Do you know about [`importorskip`](https://docs.pytest.org/en/stable/reference/reference.html#pytest-importorskip)?

Reply:
Yes. I don't think this setup is quite equivalent, though: I think the idea is that we _will_ get an error if you requested the `matplotlib` tests and it wasn't available. We use pytest marks to enable/disable this test group. Whereas if somehow matplotlib installation becomes broken/incompatible, `importorskip` would lead to a quieter skip.